contracts/standard/permission/ArbitrablePermissionList.sol

```diff
@@ -110,10 +109,10 @@ contract ArbitrablePermissionList is PermissionInterface, Arbitrable {
     /* Public */
 
     /**
-     *  @dev Request an item to be registered.
+     *  @dev Request for an item to be registered.
      *  @param _value The value of the item to register.
      */
-    function requestRegistering(bytes32 _value) public payable {
+    function requestRegisterItem(bytes32 _value) public payable {
```

Comment:
I got rid of all the `ing` endings on the function names. I also considered `requestRegistration` but thought it was better to use the same pattern as clear

Reply:
You request a `registration` or a `clearing`. So I think `requestRegistration` and `requestClearing` are more appropriate.

Reply:
:+1: